src/components/OneTimeCodeInput/index.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { OneTimeCodeInput } from './index'
+import {
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  DialogWindow,
+} from '../Dialog'
+import { Button } from '../Button'
+
+export const Basic = () => {
+  return (
+    <Dialog on>
+      {({ getWindowProps }) => (
+        <>
+          <DialogWindow {...getWindowProps()}>
+            <DialogHeader>One Time Code Input</DialogHeader>
+            <DialogBody>
+              <OneTimeCodeInput
+                length={5}
```

Comment:
Since `5` is the default, maybe nice to use `6` instead? Or add another story that shows how you can more or less than 5?